carbon-design-system/carbon
# fix(Tag): added new prop titleText to specify title on clear

Closes #4025

#### Changelog

**New**

- Added titleText prop to specify title text on clear button

Testing / Reviewing

- Go to Tab with filter
- Specify titleText prop as a string
- hover over clear button, text should be visible.
